backend/loi-service/src/main/java/org/egov/works/web/models/LetterOfIndentRequestWorkflow.java

```diff
@@ -0,0 +1,43 @@
+package org.egov.works.web.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.*;
+import org.springframework.validation.annotation.Validated;
+
+import javax.validation.Valid;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * LetterOfIndentRequestWorkflow
+ */
+@Validated
+@javax.annotation.Generated(value = "org.egov.codegen.SpringBootCodegen", date = "2022-08-04T15:05:28.525+05:30")
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class LetterOfIndentRequestWorkflow {

```

Comment:
Use workflow class from digit-models library.

Reply:
Could not find the Workflow clas in Digit-models library